openscap_report/scap_results_parser/parsers/oval_object_parser.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022, Red Hat, Inc.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from ..data_structures import OvalObject
+from .shared_static_methods_of_parser import SharedStaticMethodsOfParser
+
+MAX_MESSAGE_LEN = 99
+
+
+class OVALObjectParser:
+    def __init__(self, objects, collected_objects, system_data):
+        self.objects = objects
+        self.collected_objects = collected_objects
+        self.system_data = system_data
+
+    @staticmethod
+    def _find_item_ref(object_):
+        list_of_item_ref = [item.get('item_ref') for item in object_]
+        return list(filter(None, list_of_item_ref))
+
+    @staticmethod
+    def _complete_message(item, var_id):
+        if len(item.text) == MAX_MESSAGE_LEN and var_id[:item.text.find('(')] in var_id:
+            return f"{item.text[:item.text.find('(') + 1]}{var_id})"
+        return item.text
+
+    def _get_ref_var(self, element, xml_collected_object):
+        if xml_collected_object is None or 'var_ref' not in element.attrib:
+            return 'no value'
+        variable_value = ''
+        var_id = element.get('var_ref')
+        for item in xml_collected_object:
+            if var_id == item.get('variable_id'):
+                variable_value += item.text
```

Comment:
More pythonic way of doing this would be to collect these in a list and after that merge the items in the list using `.join()` method.